stringlength.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+)
+
+func main() {
+	stringOne := "merpflakes"
+	stringTwo := "   "
+	stringThree := ""
+
+	if len(strings.TrimSpace(stringOne)) == 0 {
```

Comment:
ok @fahdi. I will try to implement something testable with its unit tests this WE. Thanks dude 👍